cmd/rqlite/main.go

```diff
@@ -90,6 +93,14 @@ func main() {
 	})
 }
 
+func toggleTimer(op string, flag *bool) error {
+	if op != "on" && op != "off" {
+		return fmt.Errorf("not a boolean value: \"%s\". Assuming \"no\"", op)
```

Comment:
This message doesn't make sense to me. Did you mean "assuming on"? If so, why not just do that, and not return error?

Reply:
Ok, I should have mentioned that I copied the sqlite3 CLI behaviour and error message:

```
$ sqlite3
SQLite version 3.23.1 2018-04-10 17:39:29
Enter ".help" for usage hints.
Connected to a transient in-memory database.
Use ".open FILENAME" to reopen on a persistent database.
sqlite> .timer onn
ERROR: Not a boolean value: "onn". Assuming "no".
```

I can change it though.

Reply:
Yeah, probably change it.